### DISCUSSION TITLE
T2 - Duda parte 2 código base

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#138] [Tarea 2] [answered]
Hola, tengo una duda con el código base. Este debiese funcionar en un comienzo (es decir si uno no cambia nada del código, debiese correr sin errores) o tiene errores? Tengo la duda, ya que al correr el código inicial con valgrind, me sale el siguiente output:

![image](https://github.com/IIC2133-PUC/2024-1/assets/43099597/ed2cac2a-995f-460f-95b5-4c2ad5bfc2db)

Repetidas veces y no se imprimen todas las casillas del paquete (que debiese suceder, según tengo entendido)

Agradezco cualquier aclaración respecto a esto 👍

**Comment by benjavicente** [comment, marked as answer]
Hola! Probamos y nos funciona correctamente a nosotros.

Enviamos información suficiente para ver si se puede replicar, como que es lo que corres en el terminal y los archivos dados.

**Reply by MaxAl100** [reply]
El archivo que utilizo es el input.txt que se da ejemplo en el enunciado, copié línea por línea su contenido.

Ahora probé desde mi otro dispositivo (laptop) y no tuve problemas con el output; cuando tenga acceso a mi torre revisaré si existe algún error en mi archivo de input ahí, en caso contrario informaré los pasos que realicé para que me falle.

**Reply by MaxAl100** [reply]
Tenía un 2 3 en vez de un solo 3, fue la única línea donde se me quedó el marcador de línea 😅
